### PR TITLE
Support reading and writing EFI variables on Windows

### PR DESCRIPTION
**Pull request by hughsie**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [ ] Code fix
- [X] Feature
- [ ] Documentation

### CODE REVIEW

File: libfwupdplugin/fu-efivar-windows.c [hunk 1]
```diff
@@ -55,21 +95,121 @@ fu_efivar_get_data_impl(const gchar *guid,
...
+			      {FU_EFIVAR_GUID_EFI_GLOBAL, "ConOutDev"},
+			      {FU_EFIVAR_GUID_EFI_GLOBAL, "CurrentPolicy"},
+			      {FU_EFIVAR_GUID_EFI_GLOBAL, "dbDefault"},
+			      {FU_EFIVAR_GUID_EFI_GLOBAL, "dbrDefault"},
+			      {FU_EFIVAR_GUID_EFI_GLOBAL, "dbtDefault"},
+			      {FU_EFIVAR_GUID_EFI_GLOBAL, "dbxDefault"},
+			      {FU_EFIVAR_GUID_EFI_GLOBAL, "DeployedMode"},
+			      {FU_EFIVAR_GUID_EFI_GLOBAL, "DriverOrder"},
+			      {FU_EFIVAR_GUID_EFI_GLOBAL, "DriverXXXX"},
+			      {FU_EFIVAR_GUID_EFI_GLOBAL, "ErrOut"},
```
Comment (superm1) [inline libfwupdplugin/fu-efivar-windows.c:158]: you should look for the variables that fwupd makes too, no?

File: src/fu-tool.c [hunk 2]
```diff
@@ -4071,6 +4097,13 @@ main(int argc, char *argv[])
 			      /* TRANSLATORS: command description */
 			      _("Build a cabinet archive from a firmware blob and XML metadata"),
 			      fu_util_build_cabinet);
+	fu_util_cmd_array_add(cmd_array,
+			      "efivar-list",
```
Comment (superm1) [inline src/fu-tool.c:4101]: bash completion missing

Comment (hughsie) [reply src/fu-tool.c:4101]: Really? It's fwupdtool only

Comment (hughsie) [reply src/fu-tool.c:4101]: As in I thought I added it.

**Review by superm1 — APPROVED** [review]

**Pull request merged**